.github/workflows/stale.yml

```diff
@@ -38,7 +38,9 @@ jobs:
           close-issue-reason: not_planned
 
           # Labels to check for stale issues/PRs
-          any-of-labels: "needs: changes 🔁,"
+          any-of-pr-labels: "needs: changes 🔁,needs: docs 📝,needs: tests  🧪"
```

Comment:
- `needs: docs`: PR is almost ready but needs to be documented 
- `needs: tests`: PR is almost ready but unit tests are needed

So why are we close PR that is almost done? It would be a better decision to close PRs that have not been decided and discussed for a long time than those with incomplete unit testing.

Reply:
But are they really almost ready if months pass without them getting finalised?

Reply:
>  It would be a better decision to close PRs that have not been decided and discussed for a long time than those with missing unit testing.

I think when the required action is by us the team (discussion, decision), it shouldn't ever be a bot closing the PR but only us ourselves. We shouldn't shove that responsibility to a bot.

Reply:
You are right about responsibility 👍 Then I'm in favor of removing `needs: docs` and `needs: tests` labels as well from PR labels. I think these are almost complete PRs.

Reply:
Not sure...if they are around for two months without getting finalised and then after the warning by stalest nobody finalises them in the next 15 days, I think they're far from almost complete or even easy to complete. Otherwise one would manage to complete them within 60 + 15 days, right?

Reply:
If it has not been completed for a long time (for needs docs and tesst), it would be better for the team to decide this rather than throwing all the PR in the trash with a bot.

Reply:
I don't understand; If we don't put the responsibility for undecided or undiscussed uncertain PRs on the bot, why do we leave the automatic close of almost completed PRs to the bot? 🤔

Reply:
You're right